src/scripts/commands/patients.ts

```diff
@@ -69,15 +70,21 @@ const closePatientsCmd = command({
             long: "comments",
             description: "Data element of comments with its associated value DE-Value",
         }),
+        saveReportPath: option({
+            type: string,
+            long: "save-report",
+            description: "Save report to CSV file",
+        }),
         post: flag({
             long: "post",
             description: "Send payload to DHIS2 API. If not present, shows payload.",
         }),
     },
     handler: async args => {
         const api = getD2Api(args.url);
-        const programsRepository = new ProgramsD2Repository(api);
-        new ClosePatientsUseCase(programsRepository).execute(_.omit(args, ["url"]));
-        return;
+        const programsRepository = new TrackerD2Repository(api);
+        const reportExportRepository = new ReportExportCsvRepository();
+
+        new ClosePatientsUseCase(programsRepository, reportExportRepository).execute(_.omit(args, ["url"]));
```

Comment:
[subjective] No need to omit unused properties, being structural typing. Not that it's wrong to do it, though!